### ISSUE TITLE
EACCES: permission denied, open './tmp/session.json'

### ISSUE THREAD

**Issue by pr3tty8oyfl0yd** [kompot/nhl-tv-geeky-streams#36]
I am getting this however I do have a tmp and video directory and am getting the error?

EACCES: permission denied, open './tmp/session.json'

**Comment by Redmax87** [comment]
I have this error:

? Choose game to download ⤻  one day forward
? Choose game to download ⤺  one day back
? Choose game to download ⤺  one day back
? Choose game to download   Nashville Predators     @ Florida Panthers
? Choose feed to watch AWAY, FS-TN
Request failed with status code 500

**Comment by kompot** [comment]
@pr3tty8oyfl0yd 
well, are you running via docker? 
or natively? 
what platform?

@Redmax87 
have you tried other matches/feeds?
seems like temporary NHL problem, I've observed this couple of times?

**Comment by Redmax87** [comment]
It;s ok now, yaml file was the problem, were not configured correctly mail and password.
Thank you for email.

**Comment by rseanhall** [comment]
I got this error as well. I'm running on Windows with Docker Toolbox. I had downloaded the repo to `C:\src`. Apparently, Docker Toolbox does not fully support mounting volumes from the host machine into the container if the host path is outside `C:\Users`. So the `tmp` folder was mounted from the host machine, but the container could neither read nor write to it. It's worth mentioning that to even get to this state I had commented out the mapping of `.:/app` since it was failing to even run since it couldn't find `/app/package.json` due to the same permissions issue.
